webapi/src/main/scala/org/knora/webapi/store/triplestore/impl/TriplestoreServiceLive.scala

```diff
@@ -163,7 +162,14 @@ case class TriplestoreServiceLive(
     for {
       resultString <- executeSparqlQuery(query)
       _            <- ZIO.logDebug(s"sparqlHttpAsk - resultString: $resultString")
-      result       <- ZIO.attemptBlocking(resultString.parseJson.asJsObject.getFields("boolean").head.convertTo[Boolean])
+      result <-
+        ZIO
+          .fromEither(for {
+            obj     <- resultString.fromJson[Json.Obj]
+            boolean <- obj.get("boolean").toRight("Missing \"boolean\" key.")
+            result  <- boolean.asBoolean.toRight("Boolean value expected.")
+          } yield result)
+          .mapError(Throwable(_))
```

Comment:
you could probably make this a one-liner (or at least almost) by using a JsonCursor. (I have a PR open which would add documentation on that https://github.com/zio/zio-json/pull/1098)

Reply:
I hope it's fine if I leave this be.

Reply:
sure